docs/config.md

```diff
@@ -43,6 +43,305 @@ For `1x`/`2x`, initial learning rate decays by a factor of 10 at the 8/16th and
+we makea brief comments on the config of Mask R-CNN using ResNet50 and FPN as the following.
+For more detailed usage and the corresponding alternative for each modules, please refer the API documentation.
+
+```python
+model=dict(
+    type='MaskRCNN',  # The name of detector
+    pretrained='torchvision://resnet50',    # The ImageNet pretrained backbone to be loaded
+    backbone=dict(  # The config of backbone
+        type='ResNet',    # The type of the backbone, refer https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/backbones/resnet.py#L288 for more details.
+        depth=50,  # The depth of backbone, usually it is 50 or 101 for ResNet and ResNext backbones.
+        num_stages=4,  # Number of stages of the backbone, 4 for ResNet and ResNext backbones.
+        out_indices=(0, 1, 2, 3),  # The index of output feature maps produced in each stages
+        frozen_stages=1,  # The weights in the first 1 stage are fronzen
+        norm_cfg=dict(  # The config of normalization layers.
+            type='BN',  # Type of norm layer, usually it is BN of GN
+            requires_grad=True),   # Whether to train the gamma and beta in BN
+        norm_eval=True,  # Whether to freeze the statistics in BN
+        style='pytorch'),  # The style of backbone, 'pytorch' means that stride 2 layers are in 3x3 conv,
+                           # 'caffe' means stride 2 layers are in 1x1 convs.
+    neck=dict(
+        type='FPN',  # The neck of detector is FPN. We also support 'NASFPN', 'PAFPN', etc. Refer https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/necks/fpn.py#L10 for more details.
+        in_channels=[256, 512, 1024, 2048],  # The input channels, usually this is consistent with the output channels of backbone
+        out_channels=256,  # The output channels of each level of the pyramid feature map
+        num_outs=5),  # The number of output scales
+    rpn_head=dict(
+        type='RPNHead',  # The type of RPN head is 'RPNHead', we also support 'GARPNHead', etc. Refer https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/dense_heads/rpn_head.py#L12 for more details.
+        in_channels=256,  # The input channels of each input feature map, this is consistent with the output channels of neck
+        feat_channels=256,  # Feature channels of convolutional layers in the head.
+        anchor_generator=dict(  # The config of anchor generator
+            type='AnchorGenerator',  # Most of methods use AnchorGenerator, SSD Detectors uses `SSDAnchorGenerator`. Refer https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/anchor/anchor_generator.py#L10 for more details
+            scales=[8],  # Basic scale of the anchor
+            ratios=[0.5, 1.0, 2.0],  # The ratio between width and height
+            strides=[4, 8, 16, 32, 64]),  # The strides of the anchor generator. This is consistent with the FPN feature strides.
```

Comment:
We may explain the relation between anchor size and `scales`, `base_sizes` and `strides`.